src/RBACHandler.test.ts

```diff
@@ -512,7 +517,7 @@ describe('getAllowedResourceTypesForOperation', () => {
                 userIdentity: nonPractitionerDecoded,
                 operation: 'search-type',
             }),
-        ).resolves.toEqual([...financialResources, 'Patient']);
+        ).resolves.toEqual(['Patient', ...financialResources]);
```

Comment:
why was ordered change? if no reason, leave as original, so that we have less changes